### PR TITLE
Fifth task

### PR DESCRIPTION
**Pull request by an2onsol**
https://projecteuler.net/problem=5

### CODE REVIEW

File: fifth_problem.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+def smallest_multiple(first_num, second_num)
+  deviders = Array.new
```
Comment (AndreyAnischik) [inline fifth_problem.rb:2]: small type, `dividers`

File: fifth_problem.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  for num in (first_num..second_num)
+    i = 1
+    deviders.each { |d| num /= d if num % d == 0}
```
Comment (AndreyAnischik) [inline fifth_problem.rb:6]: wrong spaces in brackets

File: fifth_problem.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    deviders.each { |d| num /= d if num % d == 0 }
+    while num != 1
+      if num % i == 0
+        num /= i
+        deviders << i
+      end
+      i += 1
+    end
+  end
+  deviders.reduce(1, :*)
```
Comment (bronium) [inline fifth_problem.rb:15]: Giving an initial value here is redundant, `reduce(:*)` would give the save result

File: fifth_problem.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+def smallest_multiple(first_num, second_num)
+  deviders = []
+
+  for num in (first_num..second_num)
```
Comment (bronium) [inline fifth_problem.rb:4]: You should use `each` instead
https://rubystyle.guide/#no-for-loops

File: fifth_problem.rb [hunk 5]
```diff
@@ -0,0 +1,16 @@
+def smallest_multiple(first_num, second_num)
+  deviders = []
```
Comment (bronium) [inline fifth_problem.rb:2]: didn't fix a typo :)

**Review by bronium — APPROVED** [review]

**Pull request closed without merging**